src/org/usfirst/frc/team694/robot/FieldMapMiddleFarFromScoringTableQuadrant.java

```diff
@@ -5,14 +5,14 @@
  * All of our autons should be based on the field so that if our measurements 
  * change, we can adapt quickly. All measurements are in inches for encoder purposes.
  */
-public final class FieldMapSideClosestToScoringTableMiddleQuadrant {
+public final class FieldMapMiddleFarFromScoringTableQuadrant {
 
 /* Given FRC Manual Measurements*/
     //These distances is us starting on the left side of the field
     //all measurements are in inches or degrees if specified 
     //anything with // could be changed due to the actual field measure  
     //These are the measurements that need to be measured out before match.
 
-    public static final int SIDE_CLOSEST_TO_SCORING_TABLE_QUADRANT_DISTANCE_FROM_NULL_TERRITORY_TO_NULL_BUMP = 36; //
-    public static final double SIDE_CLOSEST_TO_SCORING_TABLE_QUADRANT_DISTANCE_FROM_BORDER_TO_SCALE_EDGE = 71.57; //
-}
\ No newline at end of file
+    public static final int MIDDLE_FAR_FROM_SCORING_TABLE_QUADRANT_DISTANCE_FROM_NULL_TERRITORY_TO_NULL_BUMP = 36; //
```

Comment:
This seems a little confusing, but as long as it makes sense to the people measuring the field.